src/meta-srv/src/metasrv/builder.rs

```diff
@@ -115,6 +117,11 @@ impl MetaSrvBuilder {
         self
     }
 
+    pub fn datanode_clients(mut self, clients: Arc<DatanodeClients>) -> Self {
+        self.datanode_clients = Some(clients);
+        self
+    }
```

Comment:
It seems that it is not necessary to configure datanode_clients, we can create a new one when creating DdlManager.

Reply:
> It seems that it is not necessary to configure datanode_clients, we can create a new one when creating DdlManager.

It's used for test purposes, in the IG tests we don't actually connect to a datanode. Instead, we configure a datanode clients.